src/EditorFeatures/TestUtilities/Workspaces/TestWorkspace_XmlConsumption.cs

```diff
@@ -445,6 +446,18 @@ private static string GetLanguage(TestWorkspace workspace, XElement projectEleme
             return languageName;
         }
 
+        private static string GetDefaultNamespace(TestWorkspace workspace, XElement projectElement)
+        {
+            var language = GetLanguage(workspace, projectElement);
+            if (language != LanguageNames.CSharp)
+            {
+                return null;
+            }
```

Comment:
this needs commenting explaining what's going on here.  also, i don't know why you actually need to check for C#.  won't this just fall out below?

Reply:
Comments added. I don't it fall through because I want the value for other language to be null instead of "" (which means global namespace)